src/applications/accreditation/21a/components/common/GetFormHelp.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 
 const GetFormHelp = () => (
-  <p>
-    <strong>If you have trouble using this online form,</strong> email VA
-    accreditation at{' '}
-    <a href="mailto:ogcaccreditationmailbox@va.gov">
-      ogcaccreditationmailbox@va.gov
-    </a>
-  </p>
+  <>
+    <p>
+      For questions about the accreditation process, visit the{' '}
+      <a href="https://www.va.gov/ogc/accreditation.asp">OGC website.</a>
+    </p>
+    <p>If you have technical trouble using this online form, ______</p>
```

Comment:
I'm assuming this will be updated?

Reply:
Yes. The v3 designs have this `______` currently, as we are waiting on the specific text from OGC. The thinking was pushing this placeholder to staging would incentivize/remind them to get it to us.